process/Code-Review/README.md

```diff
@@ -21,7 +21,8 @@ GitHub. For general Git guidelines and practices, see
       All Tech Leads are GitHub admins and will have the power to do so.
 - Don't leave pull requests open and inactive for more than 2 business days.
 - Don't merge someone else's pull request without their approval.
-- Keep "pull request fix" changes in separate commits for ease of review.
+- Keep "address code review" changes in separate commits for ease of review.
```

Comment:
[non-blocking]
Perhaps this wording is more perspicuous?
> When adding commits to a pull request that address code review comments, keep them in a separate commit for ease of review.

I was uncertain what "address code review" meant for a bit. Also, if it was meant to be an example git commit subject, perhaps it should start with a capital letter?